docs/developers_guide/Basics.md

```diff
@@ -28,24 +26,32 @@ There are following types of components:
 - `SinkFactory`
 - `CustomStreamTransformer` - types of transformations depend on type of Engine
 - `Service` - mainly for defining stateless enrichments
+To read more see [ComponentProvider API](./Components)
 
+## Execution of scenarios
```

Comment:
Execution or deployment?

Reply:
I've replaced with "Deployment of scenarios"